csv2rdf/index.html

```diff
@@ -893,13 +778,15 @@
                   the <code>name</code> property specified for each
                   <a href="http://www.w3.org/TR/tabular-data-model/#dfn-column">column</a>. 
                   During template expansion, the <em>variables</em> evaluate to the 
-                  <a href="http://www.w3.org/TR/tabular-data-model/#dfn-string-value">cell value</a>
+                  <a href="http://w3c.github.io/csvw/metadata/#dfn-cell-value">cell value</a>
                   within the 
                   <a href="http://www.w3.org/TR/tabular-data-model/#dfn-row">row</a>
                   being processed that is associated with the named 
                   <a href="http://www.w3.org/TR/tabular-data-model/#dfn-column">column</a>.</p>
                 <p>The <em>variable</em> <code>_row</code> evaluates to the number of the row being
```

Comment:
I think we need to add another variable to support the `valueUrl` property, ie `_value` evaluating to the cell value of the cell being processed.